packages/node-utils/src/http.ts

```diff
@@ -163,22 +177,42 @@ export class HttpServer<T extends EventMap> extends TypedEmitter<T & BaseEvents>
         });
     }
 
-    public post(pathname: string, handler: Handler[]) {
+    /**
+     * split /a/:b/:c
+     * to [a] and [:b, :c]
+     */
+    private splitSegments(pathname: string) {
+        const [baseSegments, paramsSegments] = arrayPartition(
+            pathname.split('/').filter(segment => segment),
+            segment => {
+                if (!segment) return true;
+                return !segment.includes(':');
+            },
```

Comment:
Thanks to the filter above, we can presume that all `segment`s are truthy, right?
```suggestion
            segment => !segment.includes(':'),
```

Reply:
[9e5ad43](https://github.com/trezor/trezor-suite/pull/9985/commits/9e5ad43d856d6d2dbed31bb60e1177bb7210f7ac)